package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "videojs-contrib-hls",
-  "version": "1.3.3",
+  "version": "1.3.4",
```

Comment:
This is a bit confusing. Did you forget to bump the package.json in the last release?